pkgs/development/python-modules/pytorch3d/default.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  torch,
+  iopath,
+  cudaPackages,
+  config,
+  cudaSupport ? config.cudaSupport,
+}:
+
+assert cudaSupport -> torch.cudaSupport;
+
+buildPythonPackage rec {
+  pname = "pytorch3d";
+  version = "0.7.8";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "facebookresearch";
+    repo = "pytorch3d";
+    rev = "V${version}";
+    hash = "sha256-DEEWWfjwjuXGc0WQInDTmtnWSIDUifyByxdg7hpdHlo=";
+  };
+
+  nativeBuildInputs = lib.optionals cudaSupport [ cudaPackages.cuda_nvcc ];
+  build-system = [
+    setuptools
+    wheel
```

Comment:
```suggestion
```

Reply:
H'm. It came from nix-init, it's probably declared in the toml?